hedgehog-extras.cabal

```diff
@@ -80,19 +86,25 @@ library
                         directory,
                         exceptions,
                         filepath,
+                        generic-lens,
                         hedgehog,
                         http-conduit,
+                        hw-prelude,
                         lifted-async,
                         lifted-base,
+                        microlens,
                         monad-control,
                         mmorph,
                         mtl,
                         network,
+                        primitive,
```

Comment:
this dependency looks unused

Reply:
I was used, but I've removed the code that used it, so this can be deleted.